src/ngraph/ngraph_sgcompiler_utils.h

```diff
@@ -83,6 +83,9 @@ inline ngraph::Shape TShape_to_NShape(const nnvm::TShape& inshape) {
   return convert_shapes<nnvm::TShape, ngraph::Shape>(inshape);
 }
 
+// FIXME: Can we replace all of the 'makeConstant' machinery below with
+// calls to 'ngraph::op::Constant::create(...)'?
```

Comment:
We don't want to create large constants arrays in memory directly, the scalar->broadcast mechanism is still easier to optmize, and removing these utilities leads to a lot of code copy for extract shape/ extract data type / create constant/ broadcast